src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -292,29 +345,29 @@ try {
 
 ### Polyfill
 
-It is not possible to polyfill the Multi-Screen Window Placement API, but you
-can shim its shape so you can code exclusively against the new API:
+It is not possible to polyfill the Multi-Screen Window Placement API, but you can shim its shape so
+you can code exclusively against the new API:
 
 ```js
-if (!("getScreens" in window)) {
+if (!('getScreens' in window)) {
   // Returning a one-element array with the current screen,
   // noting that there might be more.
   window.getScreens = async () => [window.screen];
-  // Returning `false`, noting that this might be a lie.
-  window.isMultiScreen = async () => false;
+  // Set to `false`, noting that this might be a lie.
+  window.screen.isExtended = false;
 }
 ```
 
-The other aspects of the API—the `onscreenschange` event and the `screen` property of the
-`FullscreenOptions`—would simply never fire or silently be ignored respectively by non-supporting
+The other aspects of the API, that is, the `screenschange` event and the `screen` property of the
```

Comment:
nit: "that is, the screen change events and" (there are multiple events that wouldn't fire)

Reply:
Gotcha.